lib/miq_helper.rb

```diff
@@ -0,0 +1,26 @@
+require 'pathname'
+require 'active_support/core_ext/object/blank'
+require 'active_support/string_inquirer'
+
+module Miq
```

Comment:
Well, I went with `lib/miq_helper.rb` because I was matching `lib/vmdb_helper.rb`.  Fine with changing it if you want.  Just wanted to keep it as short as possible as well (otherwise, people will be less inclined to use it 😉 )